docs/basic-setup.md

```diff
@@ -156,24 +156,28 @@ There are two possible reasons for the authentication server to reject a request
 - Provided token is expired or invalid (HTTP response 401 `Unauthorized`)
 - Provided token is valid but scopes are insufficient (HTTP response 403 `Forbidden`)
 
-## Clearing expired access & refresh tokens
+## Clearing expired access, refresh tokens and auth codes
 
-To clear expired access & refresh tokens you can use the `trikoder:oauth2:clear-expired-tokens` command.
+To clear expired access, refresh tokens and auth codes you can use the `trikoder:oauth2:clear-expired-tokens` command.
```

Comment:
@X-Coder264 Keeping `and` instead of adding a comma makes more sense to me tbh.

Reply:
@HypeMC Fixed.